src/relay/backend/compile_engine.cc

```diff
@@ -83,11 +83,18 @@ class ScheduleGetter :
     cache_node->func_name = readable_name_stream_.str();
     CachedFunc cfunc(cache_node);
     CHECK(master_op_.defined());
+    // Filter PlaceholderOp from the outputs for schedule.
```

Comment:
Can we add a more descriptive comment here? for example, explaining that fusion over tupled results may leave identity relationships between inputs and outputs, and those should not be scheduled.